cypress/integration/pages/liveRadio.js

```diff
@@ -0,0 +1,245 @@
+            dir: `${appConfig[service].dir}`,
+          });
+        });
+      });
+
+      it.skip('should have the correct facebook metadata', () => {
+        cy.checkFacebookMetadata(
+          '100004154058350',
+          '1609039196070050',
+          `${appConfig[service].articleAuthor}`,
+        );
+      });
+
+      it.skip('should have the correct open graph metadata', () => {
+        cy.checkOpenGraphMetadata(
+          'Meghan follows the royal bridal tradition started by the Queen Mother in 1923.',
+          `${appConfig[service].defaultImage}`,
+          `${appConfig[service].defaultImageAltText}`,
+          `${appConfig[service].locale}`,
+          `${appConfig[service].defaultImageAltText}`,
+          "Meghan's bouquet laid on tomb of unknown warrior",
+          'article',
+          `https://www.bbc.com${config[service].pageTypes.liveRadio.path}`,
+        );
+      });
+
+      it.skip('should have the correct twitter metadata', () => {
+        cy.checkTwitterMetadata(
+          'summary_large_image',
+          `${appConfig[service].twitterCreator}`,
+          'Meghan follows the royal bridal tradition started by the Queen Mother in 1923.',
+          `${appConfig[service].defaultImageAltText}`,
+          `${appConfig[service].defaultImage}`,
+          `${appConfig[service].twitterSite}`,
+          "Meghan's bouquet laid on tomb of unknown warrior",
+        );
+      });
+
+      it.skip('should include metadata that matches the JSON data', () => {
+        cy.request().then(({ body }) => {
+          cy.get('head').within(() => {
+            cy.get('meta[name="description"]').should(
+              'have.attr',
+              'content',
+              body.pageData.promo.summary || body.pageData.promo.name,
+            );
+            cy.get('meta[name="og:title"]').should(
+              'have.attr',
+              'content',
+              body.pageData.promo.name,
+            );
+            cy.get('meta[name="og:type"]').should(
+              'have.attr',
+              'content',
+              body.pageData.metadata.type,
+            );
+          });
+          cy.get('html').should(
+            'have.attr',
+            'lang',
+            body.pageData.metadata.passport.language,
+          );
+        });
+      });
+
+      // will be addressed by this https://github.com/bbc/simorgh/issues/3117
+      it.skip('should include mainEntityOfPage in the LinkedData', () => {
+        cy.get('script[type="application/ld+json"]')
+          .should('contain', 'mainEntityOfPage')
+          .and('contain', 'headline');
+      });
+    });
+
+    describe('Scripts', () => {
+      it('should only have expected bundle script tags', () => {
+        cy.hasExpectedJsBundles(envConfig.assetOrigin, service);
+      });
+    });
+  });
+
+const runCanonicalTests = ({ service }) =>
+  describe('Canonical Tests', () => {
+    // will be addressed by https://github.com/bbc/simorgh/pull/2971
```

Comment:
```suggestion
    // will be addressed by https://github.com/bbc/simorgh/issues/3324
```

Reply:
good point I'll unskip those tests and see what happens!